extension/flagMapProducts.js

```diff
@@ -0,0 +1,31 @@
+module.exports = function (context, input, cb) {
+  const triggerProps = [
+    {
+      label: 'Apply Map',
```

Comment:
The `label` of the apply map have a different capatalization like: `Apply MAP` same goes for `value`.

You can make sure that they fitting by just make the label and the value lowercase by a function and then compare it

Reply:
made comparison of map label and value so it is case insensitive.